pack/mojo_encounter.json

```diff
@@ -454,7 +455,7 @@
 		"quantity": 1,
 		"set_code": "spiral",
 		"set_position": 5,
-		"text": "Permanent.\n<b>Forced Interrupt</b>: After the last threat is removed from here, the player who removed that threat reveals the top card of the show deck and places 3 [per_hero] threat here.\n<b>Hero Action</b>: Take 2 damage. Remove 3 threat from here.",
+		"text": "Permanent.\n<b>Forced Interrupt</b>: After the last threat is removed from here, the player who removed that threat reveals the top card of the show deck and places 3 [per_hero] threat here.\n<b>Hero Action</b>: Take 2 damage → Remove 3 threat from here.",
```

Comment:
1. I think it should now read `Take 2 damage → remove 3 threat from here.`
2. Existing issue but I don't think we need the space with `3 [per_hero]`. I think it should be `3[per_hero]`.